plugins/elanfp/fu-elanfp-file-control.h

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (C) 2021 Michael Cheng <michael.cheng@emc.com.tw>
+ *
+ * SPDX-License-Identifier: LGPL-2.1+
+ */
+
+#pragma once
+
+//#include <glib.h>
```

Comment:
this looks like it can be dropped

Reply:
Fixed in 331c4c3c91fb57511500e163d4596d23da26e1b2